utilities/Makefile.am

```diff
@@ -0,0 +1 @@
+SUBDIRS = flex_counter debug_counter
```

Comment:
>flex_counter [](start = 10, length = 12)

Where is this?

Reply:
Oops, I forgot to remove this when I split that into #1093